src/openeo_test_suite/tests/processes/processing/test_example.py

```diff
@@ -74,22 +89,30 @@ def test_process(
     # prepare the arguments from test JSON encoding to internal backend representations
     # or skip if not supported by the test runner
     try:
-        arguments = prepare_arguments(example["arguments"], id, connection, file)
+        arguments = _prepare_arguments(
+            arguments=example["arguments"],
+            process_id=process_id,
+            connection=connection,
+            file=file,
+        )
     except Exception as e:
-        # TODO: skipping on a generic `Exception` is very liberal and might hide real issues
```

Comment:
I can only check what I meant once I can run it again ;-)

Reply:
I'm going through it now to check what changes if we change to NotImplementedError and if the additional failures are test issues etc. Stay tuned ;-)

Reply:
I just pushed https://github.com/Open-EO/openeo-test-suite/pull/21/commits/7b28019f4ec43548ce70ed09ff71b56dfc7130d1 which switches to only NotImplementedError and fixes a couple of things in the background so that test coverage is still similar.